app/models/identity_tijuana/donation.rb

```diff
@@ -0,0 +1,86 @@
+module IdentityTijuana
+  class Donation < ApplicationRecord
+    include ReadWrite
+    self.table_name = 'donations'
+    belongs_to :user
+    has_many :transactions
+
+    scope :updated_donations, -> (last_updated_at, exclude_from) {
+      where('donations.updated_at > ?', last_updated_at)
+        .and(where('donations.updated_at < ?', exclude_from))
+        .order('donations.updated_at')
+        .limit(Settings.tijuana.pull_batch_amount)
+    }
+
+    scope :updated_donations_all, -> (last_updated_at, exclude_from) {
+      where('donations.updated_at > ?', last_updated_at)
+        .and(where('donations.updated_at < ?', exclude_from))
+    }
+
+    def self.import(donation_id, sync_id)
+      donation = Donation.find(donation_id)
+      donation.import(sync_id)
+    end
+
+    def import(sync_id)
+      member = Member.find_by_external_id(:tijuana, user_id)
+      if member.present?
+        if member.ghosting_started?
```

Comment:
This true for both when ghosting has started and after it's finished? (badly named property if so!)

Reply:
Yep. Badly named property.